spec/fortnox/api/repositories/examples/find_examples.rb

```diff
@@ -6,11 +9,15 @@
   end
 
   describe '.find' do
+    include_context 'JSONHelper'
+
     specify 'returns correct class' do
       expect( find_id_1.class ).to be described_class::MODEL
     end
 
     specify 'returns correct Customer' do
+      id_attribute_json = subject.options.unique_id
+      id_attribute = JSONHelper.new.convert_from_json(id_attribute_json)
```

Comment:
Add `module_function :method` after the declaration of the methods you want to use from the module instead. That will allow you to do `Fortnox::API::Repository::JSONConvertion.convert_from_json( id_attribute_json )` directly without having to wrap it in a new class and instantiating it.

See http://apidock.com/ruby/Module/module_function for docs :)